intern/opensubdiv/gpu_shader_opensubdiv_geometry.glsl

```diff
@@ -47,6 +47,10 @@ layout(triangle_strip, max_vertices = 4) out;
   #extension GL_EXT_texture_buffer_object: enable
 #endif
 
+#if __VERSION__ < 130
```

Comment:
Is this modification useful if OpenSubdiv is disabled ?

Reply:
I have to test if it compiles ok without it